dist/i18n-react.d.ts

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-export declare type NotFound = string | ((key: string, context?: any) => string);
+export declare type NotFound = string | ((key: string, context?: any) => any);
```

Comment:
*React.Node* is better then *any*

Reply:
But what if users have already declared `notFound` functions which returns plain strings?